static/app/views/releases/list/releasesAdoptionChart.tsx

```diff
@@ -114,7 +114,7 @@ class ReleasesAdoptionChart extends Component<Props> {
     }));
   }
 
-  handleClick = (params: {seriesId: string}) => {
+  handleClick = params => {
```

Comment:
Did this revert to a `any` type?

Reply:
Ah good call, I thought it would infer that it is a `EChartClickHandler`